Source/Core/DolphinWX/ISOFile.cpp

```diff
@@ -135,88 +132,92 @@ GameListItem::GameListItem(const std::string& _rFileName, const Core::TitleDatab
   else
   {
     // Volume banner. Typical for everything that isn't a DOL or ELF.
-    SetWxBannerFromRaw();
+    SetWxBannerFromRaw(m_banner);
   }
 }
 
 bool GameListItem::IsValid() const
```

Comment:
Is it worthwhile to merge these boolean statements into the return?  How much is "IsValid" hit?